config/routes.rb

```diff
@@ -6,6 +6,9 @@
   namespace :questions do
     get '/start', to: 'start#index', as: 'start'
     post '/start', to: 'start#submit', as: 'start_submit'
+
+    get '/describe-repair', to: 'describe_repair#index', as: 'describe_repair'
+    post '/describe-repair', to: 'describe_repair#submit'
```

Comment:
Does this belong under the 'questions' path, or is it its own thing? Maybe it doesn't matter at this point - easy to change later?

Reply:
Possibly. As this is a named route it'd be easy to change in the future.

Reply:
On second thoughts, I agree with you. I'll move these out of the 'questions' namespace.